map-data/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'java-library'
```

Comment:
Didn't expect this ^^
Are you planning to migrate some of the existing project to use this plugin, or just for this particular project?

Reply:
No plans per-say. There was recently (now removed) a reader class in `map-data` whereby you'd always need the `xml-reader` sub-project. Now that is removed and `map-data` and `xml-reader` are decoupled. At this point having this as a `java-library` does not quite make as much sense. With that said, there are places where I think it would be of benefit to use `java-library`, notably anywhere if you use one dependency import then you would always need another.